docs/modules/device-lab-management/pages/android-devices/prepare-chinese-android-device.adoc

```diff
@@ -0,0 +1,63 @@
+= Prepare a Chinese Android device for hosting
+:navtitle: Prepare a Chinese Android device
+
+Learn how to prepare a **Chinese Android device** to be utilized with Kobiton 4.0 and later.
```

Comment:
Remove this line as the following lines more or less have the same idea.

Reply:
Line removed